scripts/release/prepare-canary-commands/download-build-artifacts.js

```diff
@@ -20,6 +20,10 @@ const run = async ({build, cwd}) => {
     entry => entry.path === 'home/circleci/project/node_modules.tgz'
   ).url;
 
+  if (!existsSync(join(cwd, 'build'))) {
+    await exec(`mkdir ./build`, {cwd});
+  }
```

Comment:
This change (and the one in `check-out-packages`) was just fixing an edge case I noticed while testing these changes.